core/src/scss/components/nav-toggle/_nav-toggle.scss

```
@@ -0,0 +1,67 @@
+@charset 'UTF-8';
+//
+// Navigation Toggle
+//
+// A clickable drop button as an alternate to an accordian style menu.
+//
+// .su-nav-toggle--light - A lighter option.
+//
+// Markup: ../templates/components/nav-toggle/nav-toggle.twig
+//
+// Style guide: Components.Nav Toggle
+//
+
+.su-nav-toggle {
+  @include margin(0);
+  @include padding(15px 28px);
+  vertical-align: middle;
+  text-align: center;
+  font-size: 0;
+  text-indent: -9999px;
+  overflow: hidden;
+  width: 49px;
+  max-width: 49px;
+  height: 58px;
+  line-height: 58px;
+  position: absolute;
+  right: 0;
+  top: 0;
+  z-index: 10;
+  background: transparent;
+  outline: none;
+  box-shadow: none;
+
+  &::before {
+    content: "";
+    background: color(border--expanded, $su-nav-colors) url("#{$su-image-path}/caret-down-white.svg") no-repeat center / 50%;
```

```suggestion
    background: color(border--expanded, $su-secondary-nav-colors) url("#{$su-image-path}/caret-down-white.svg") no-repeat center / 50%;
```
This should be bright red